src/types.ts

```diff
@@ -49,10 +49,29 @@ export type ZodValidationErrorDetail = {
   message: string;
 };
 
+export type StandardResponseInput = {
+  success: boolean;
+  data: object;
+  message?: string;
+};
```

Comment:
consider using generic type here

```suggestion
export type StandardResponseInput<TData> = {
  success: boolean;
  data: <TData>;
  message?: string;
};
```

Reply:
Im not sure generics is what we want here, and theres really no reason for us to be type checking it other than it being an object.